ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -762,7 +767,7 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
     free(messageMemory);
 }
 
-static PrjFS_Result HandleEnumerateDirectoryRequest(const MessageHeader* request, const char* relativePath)
+static PrjFS_Result HandleEnumerateDirectoryRequest(const MessageHeader* request, const char* fullPath, const char* relativePath)
```

Comment:
In the signature we use absolutePath not fullPath.
The existing convention for the file appears to be fullPath.  I prefer absolutePath myself but may be worth sticking to fullPath to avoid larger changes in this PR.

Reply:
Yeah, I feel `absolutePath` is clearer ("full" also is a possible state for a VFS4G-tracked file) but I've kept `fullPath` where there already was a variable of that name.

Reply:
@pmj my vote would be to use fullPath for this PR, enter an issue to update all fullPath's to absolute path, and we can address in a separate PR.  Am I correct in saying every place we use 'fullPath' in this file we could use 'absolutePath' instead?